option optimize_for = LITE_RUNTIME;

message CrusadeFortressRewardRequest
{
 required uint32 id = 1;
}
